hack/make/.integration-daemon-start

```diff
@@ -16,6 +16,10 @@ if [ "$(go env GOOS)" = 'windows' ]; then
        return
 fi
 
+if [ ! -x "/etc/docker/seccomp.json" ]; then
+	install_seccomp "$base/../../profiles/seccomp/default.json"
```

Comment:
Yeah, this isn't safe to include here (since these scripts are supposed to be runnable on your host, too).  Couldn't/shouldn't the default profile path be a daemon flag/config option? (esp. so that the tests could point directly to the copy in the repo)